packages/x-components/src/x-installer/api/api.types.ts

```diff
@@ -135,6 +135,21 @@ export interface SnippetConfig {
   filters?: string[]
   /** List of queries to preview. */
   queriesPreview?: QueryPreviewInfo[]
+  /** Selector for the container where the search box element will be embedded with the teleport feature. */
+  searchBoxSelector?: string
+  /** Selector for the container where the results element will be embedded with the teleport feature. */
+  resultsSelector?: string
+  /** Selector of the element the search layer is positioned right below. */
+  layerSelector?: string
+  /**
+   * View mode of the search experience.
+   *
+   * fullScreen: The search experience is displayed as a full screen modal, this mode includes the
+   * possibility to specify a layerSelector.
+   * embedded: The search experience consists of components embedded in the customer's website.
```

Comment:
```suggestion
   * embedded: The search experience consists of components embedded in a web client
```

Enforce the meaning of 'una web cliente' as an architecture approach, instead of 'la web del cliente'. No se si me explico 😅